torch_ipex/csrc/jit/fusion_pass.cpp

```diff
@@ -198,7 +199,8 @@ class OpFuser {
     }
 
     // throw
-    auto er = script::ErrorReport(node->sourceRange());
+    //auto er = script::ErrorReport(node->sourceRange());
```

Comment:
If the code is invalid, could we remove it?

Reply:
done.